dissect/target/plugins/os/unix/iptables.py

```diff
@@ -0,0 +1,146 @@
+import re
+from datetime import datetime
+from pathlib import Path
+from typing import Iterator
+
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+from dissect.target.target import Target
+
+PATTERN_IPTABLES_SAVE_GENERATED = re.compile(r"# Generated by (ip6?tables-save) v([.\d]+) on (.+)")
+
+PATTERN_IPTABLES_SAVE_POLICY = re.compile(
+    r":(?P<chain>[\w-]+) (?P<policy>[\w-]+|-) \[(?P<packet_count>\d+):(?P<byte_count>\d+)]"
+)
+
+PATTERN_IPTABLES_SAVE_RULE = re.compile(
+    r"(?:\[(?P<packet_count>\d+):(?P<byte_count>\d+)] )?(?P<rule>-A (?P<chain>[\w-]+).+)"
```

Comment:
```suggestion
    r"(?:\[(?P<packet_count>\d+):(?P<byte_count>\d+)] )?(?P<rule>-A (?P<chain>\S+).+)"
```

Otherwise, you cannot parse:

`-A TEST! -s 9.9.9.9/32 -j DROP`